src/main/webapp/dev/js/main/googleAnalytics.es6

```diff
@@ -9,7 +9,7 @@ _gaq.push(['_trackPageview']);
     ga.async = true;
     // Always use the ssl version, if not test will fail as local testing uses non HTTPs by default
     ga.src = 'https://ssl.google-analytics.com/ga.js';
-    const scripts = document.getElementsByTagName('script');
+    const scripts = $('script');
```

Comment:
I just realized that the current code does not allow this particular replacement. My apologies, can you revert this one?